app/models/compute_resources/foreman/model/ovirt.rb

```diff
@@ -30,6 +30,42 @@ def supports_update?
       true
     end
 
+    def supports_operating_systems?
+      client.respond_to?(:operating_systems) || client.send(:client).respond_to?(:operating_systems)
```

Comment:
Due to fog not exposing this feature and compatibility issues between latest foreman, fog and rbovirt, I've added a variant to be able to use the rbovirt directly. Refactored a bit to make it cleaner what `clinet.send(:client)` means and why. I'm going to open a fog PR after the rbovirt part is released.